tame.lua

```diff
@@ -0,0 +1,38 @@
+-- tame and train animals {set animals as Wild through to Masterfully Trained}
+
+local help=[====[
+tame
+====
+Tame and train animals.
+
+Usage: tame -set <level>
+
+:0: wild
+:1: trained
+:2: well-trained
+:3: skillfully trained
+:4: expertly trained
+:5: exceptionally trained
+:6: masterfully trained
+:7: tame
+:8: semi-wild
+]====]

```

Comment:
Looking at the animal_training_level enum (I got this from df/animal_training_level.h, but you could also use `lua @df.animal_training_level` for similar output):
```
        SemiWild, // 0, 0x0
        Trained, // 1, 0x1
        WellTrained, // 2, 0x2
        SkilfullyTrained, // 3, 0x3
        ExpertlyTrained, // 4, 0x4
        ExceptionallyTrained, // 5, 0x5
        MasterfullyTrained, // 6, 0x6
        Domesticated, // 7, 0x7
        Unk8, // 8, 0x8
        /**
         * Seems to be used as default when not flags1.tame
         */
        WildUntamed, // 9, 0x9

```
so I think your names for at least `0` and `8` might be wrong here, unless the enum values have changed in DF. Can you double-check those?

Reply:
Yea I'll change things up, I was only trying to match the order of this table as closely as possible. Although now that I think about it, it certainly would make more sense to have semi-wild be at 0. Though I'm not sure what to do with `Unk8` should I just use it as Wild, or both 8 and 9 as Wild =/
![image](https://user-images.githubusercontent.com/7276349/47064044-97fa6500-d192-11e8-81b0-cc06ed63852d.png)

Reply:
Oh, yeah, the wiki table is probably in a more human-friendly order.